src/algebra/group_power/basic.lean

```diff
@@ -658,6 +658,30 @@ by simpa only [sqr_abs] using pow_le_pow_of_le_left (abs_nonneg x) h 2
 theorem sqr_le_sqr' (h1 : -y ≤ x) (h2 : x ≤ y) : x ^ 2 ≤ y ^ 2 :=
 sqr_le_sqr (abs_le.mpr ⟨h1, h2⟩)
 
+theorem abs_lt_abs_of_sqr_lt_sqr (h : x^2 < y^2) : abs x < abs y :=
+lt_of_pow_lt_pow 2 (abs_nonneg y) $ by rwa [← @sqr_abs _ _ x, ← @sqr_abs _ _ y] at h
```

Comment:
The fact  you need the `@` here suggests that `sqr_abs` ought to have explicit `x` and `y` arguments.

Reply:
Okay, I'll make that change.